job_templates/power_action_-_ansible_default.erb

```diff
@@ -0,0 +1,28 @@
+<%#
+name: Power Action - Ansible Default
+job_category: Power
+description_format: "%{action} host"
+snippet: false
+template_inputs:
+- name: action
+  required: true
+  input_type: user
+  description: Action to perform on the service
+  options: "restart\r\nshutdown"
+  advanced: false
+provider_type: Ansible
+kind: job_template
+model: JobTemplate
+%>
+
+---
+- hosts: all
+  tasks:
+    - command: |
+        echo <%= input('action') %> host && sleep 3
+        <%= case input('action')
+          when 'restart'
+            'reboot'
```

Comment:
@ekohl It's less consistent with the equivalent SSH template - what about changing that in both templates after it's merged if it's important?

Reply:
Consistency with other templates is more important, but I think systemd wants to deprecate ```halt``` and ```reboot``` in favor of ```shutdown``` so it might be good to pick this up in a separate PR.

Reply:
Looks like I was not the only one: https://github.com/theforeman/community-templates/pull/456